unity/Assets/Scripts/ARFlow/ARFlowClient.cs

```diff
@@ -38,30 +38,22 @@ public ARFlowClient(string address)
 
         /// <summary>
         /// Connect to the server with a request that contain register data of about the camera.
+        /// This will throw an error if the connection is not established.
         /// </summary>
         /// <param name="requestData">Register data (AKA metadata) of the camera. The typing of this is generated by Protobuf.</param>
-        public void Connect(RegisterRequest requestData)
+        public void Connect(RegisterClientRequest requestData)
         {
-            try
-            {
-                var response = _client.register(requestData);
-                _sessionId = response.Uid;
+            var response = _client.RegisterClient(requestData);
+            _sessionId = response.Uid;
 
-                Debug.Log(response.Uid);
-            }
-            catch (Exception e)
-            {
-                // Try to catch any exceptions.
-                // Network, device image, camera intrinsics
-                Debug.LogError(e);
-            }
+            Debug.Log(response.Uid);
```

Comment:
I think we should not have `Debug.Log` in our core library. This logging call is slowing down the overall performance quite a bit. Maybe we can set up a verbose parameter or something to enable/disable the logging based on needs.

Reply:
Sounds good